OxfordBodleian/Juel-Jensen/BDLaethe30.xml

```diff
@@ -580,8 +581,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </collation>
                                 <condition key="good">                   
                                     The textblock is affected with humidity, especially towards the end of the manuscript.
```

Comment:
Affected by? What do you think, @Gnisci ?

Reply:
I am also not sure about the verbe and the tense. Also I think this damage affects the whole parchment not just the text block, so the whole sentence sounds a bit odd to me. "Humidity appears to have damaged the parchment, especially towards the end of the manuscript" or "Many of the pages have been affected/damaged by humidity, especially those at the end of the manuscripts"?